### ISSUE TITLE
Building docs fails

### ISSUE THREAD

**Issue by milancurcic** [Cloud-Drift/clouddrift#80]
See https://github.com/Cloud-Drift/clouddrift/actions/runs/3935783550/jobs/6731763496.

The relevant bit is:

```
Theme error:
An error happened in rendering the page api.
Reason: UndefinedError("'logo' is undefined")
make: *** [Makefile:20: html] Error 2
Error: Process completed with exit code 2.
```

This error goes away for me locally after I have commented out the `html_theme_options` in docs/conf.py. However, it doesn't seem to go away in GitHub Actions and I don't understand why.

@philippemiron do you have an idea?

**Comment by philippemiron** [comment]
I haven't test but I believe it's fixed in >=6.0.0. There is a workaround in this thread, [here](https://github.com/pydata/pydata-sphinx-theme/issues/1094#issuecomment-1368264928).

**Comment by milancurcic** [comment]
Indeed, I can confirm that it worked for me locally only because I was on an older Sphinx (5.3.0). Updating it to 6.0.0 broke it. The workaround that you link to seems to work; it's unclear to me from the rest of the thread and the linked PR what needs to be changed in conf.py. So I'll use the workaround instead for now.
